website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx

```diff
@@ -155,7 +157,41 @@ If you've set `enableAutoEncrypt: true`, this is also supported.
 With the above settings added to your existing config, follow the [Upgrading](/docs/k8s/operations/upgrading)
 guide to upgrade your cluster and then come back to the [Federation Secret](#federation-secret) section.
 
--> **NOTE:** You must be using consul-helm 0.21.0+. To update, run `helm repo update`.
+-> **NOTE:** You must be using consul-helm 0.21.0+.
+
+#### ProxyDefaults
+
+If you are using consul-helm 0.30.0+ you must also create a [`ProxyDefaults`](/docs/connect/config-entries/proxy-defaults)
+resource to configure Consul to use the mesh gateways for service mesh traffic.
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ProxyDefaults
+metadata:
+  name: global
+spec:
+  meshGateway:
+    mode: 'local'
+```
+
+The `spec.meshGateway.mode` can be set to `local` or `remote`. If set to `local`,
+traffic from one datacenter to another will egress through the local mesh gateway.
+If set to `remote`, traffic will be routed directly to the remote mesh gateway
+(resulting in one less hop).
```

Comment:
Why would you use local if remote is one less hop? (I think I'm missing some context in functionality, maybe that's worth adding here?)

Reply:
I suspect from a networking perspective, you can control where network traffic is allowed to exit the cluster so only service mesh traffic actually ever gets out of a kubernetes cluster.
But this is a good callout. Makes sense to add it.

Reply:
Added:

> This may be useful if you prefer all your cross-cluster network traffic to egress
from the same locations.

Reply:
Ahh got it, thanks!